screens/LoginScreen.tsx

```diff
@@ -14,19 +15,20 @@ import Colors from '../theme/Colors';
 
 import { StyledView, StyledWideContainer } from './LoginScreen.styled';
 
-const LoginSchema = Yup.object().shape({
-  email: Yup.string()
-    .email('Invalid email')
-    .required('Required'),
-  password: Yup.string()
-    .min(6, 'Too Short!')
-    .required('Required'),
-});
-
 type Props = NavigationData<'Login'>;
 
 const LoginScreen: React.FC<Props> = () => {
   const dispatch = useDispatch();
+  const { t } = useTranslation();
+
+  const LoginSchema = Yup.object().shape({
+    email: Yup.string()
+      .email(t('Invalid email'))
```

Comment:
I see that those translations are lacking in PL translations.

Reply:
So, as I have written in the PR's description I added just a sample of translations. What I focused here was to wrap every string with the `t` function. Of course we will add another language in the future but now I would recommend to focus on main features in order not to lose motivation :laughing:

Reply:
Good, I understand this :smile: Then it can stay like it is for now.